distribution/docs/src/main/resources/content/_metadataAttributes/NITF_Header-table.adoc

```diff
@@ -134,33 +134,29 @@ else current time
 |String
 |true
 
-// TODO CAL-348 Document NITF to DDF Catalog Taxonomy Security Mappings
-//|NitfHeader.FSCTLH
-//|TODO
-//|security.dissemination-controls
-//|String
-//|true
-
-// TODO CAL-348 Document NITF to DDF Catalog Taxonomy Security Mappings
-//|NitfHeader.FSCTLH
-//|TODO
-//|ext.nitf.file-control-and-handling
-//|String
-//|true
-
-// TODO CAL-348 Document NITF to DDF Catalog Taxonomy Security Mappings
-//|NitfHeader.FSREL
-//|TODO
-//|security.releasability
-//|String
-//|true
-
-// TODO CAL-348 Document NITF to DDF Catalog Taxonomy Security Mappings
-//|NitfHeader.FSREL
-//|TODO
-//|ext.nitf.file-releasing-instructions
-//|String
-//|true
+|NitfHeader.FSCTLH
+|
```

Comment:
Do we need to fill out the `Translation` column on these entries? Looks like other ones just put `none`.